x-pack/plugins/session_view/public/hooks/use_date_format.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useUiSetting } from '@kbn/kibana-react-plugin/public';
+
+export const useDateFormat = (): string => useUiSetting<string>('dateFormat');
```

Comment:
we avoid using hardcoded values directly, so I would only add a const here on this file:

```js
const DEFAULT_DATE_FORMAT = 'dateFormat';
```

and then use like that:

```js
export const useDateFormat = (): string => useUiSetting<string>(DEFAULT_DATE_FORMAT);
```